example_api.conf

```diff
@@ -0,0 +1,21 @@
+[cyberwatch]
+api_key = ZYZO...
+secret_key = Z9ULJVnqZWbfxDxOUsn...
+url = https://[CYBERWATCH_URL]
+
+[splunk]
```

Comment:
I don't think we should keep these examples here if we do not implement the features for powershell yet.